_includes/footer.html

```diff
@@ -130,4 +130,4 @@
 <!-- our external javascript scripts -->
 <script type="text/javascript" src="/assets/statics/polyfills/fetch.js"></script>
 <script type="text/javascript" src="/assets/statics/polyfills/template.js"></script>
-<script type="text/javascript" src="/assets/statics/polyfills/jsonp.js"></script>
\ No newline at end of file
+<script type="text/javascript" src="/assets/statics/polyfills/jsonp.js"></script>
```

Comment:
Is probably just the way Github shows how happy he is to have a new empty line at the end of the file